app/views/active_storage/blobs/_blob.html.erb

```diff
@@ -1,6 +1,6 @@
 <figure class="attachment attachment--<%= blob.representable? ? "preview" : "file" %> attachment--<%= blob.filename.extension %>">
   <% if blob.representable? %>
-    <%= image_tag blob.representation(resize_to_limit: local_assigns[:in_gallery] ? [ 800, 600 ] : [ 1024, 768 ]) %>
+    <%= image_tag Rails.application.routes.url_helpers.rails_representation_url(blob.representation(resize_to_limit: local_assigns[:in_gallery] ? [ 800, 600 ] : [ 1024, 768 ]).processed, host: ENV['APP_HOST']) %>
```

Comment:
@alis-khadka can you provide some documentation behind this fix? Im trying to understand this call to `processed`

``` ruby
[ 1024, 768 ]).processed
```

Reply:
Sure, I will update you.

Reply:
https://stackoverflow.com/a/53547638/6931950

Here is the stackoverflow page that I followed.
@donrestarone